opensrp-pnc/src/main/java/org/smartregister/pnc/presenter/PncProfileOverviewFragmentPresenter.java

```diff
@@ -106,12 +103,24 @@ private void generateYamlConfigList(@NonNull Facts facts, @NonNull List<YamlConf
             if (valueCount > 0) {
                 yamlConfigListGlobal.addAll(yamlConfigList);
             }
+
+            if ("live_births".equals(yamlConfig.getSubGroup())) {
+                addLiveBirths(facts.get("base_entity_id"), yamlConfigListGlobal, facts);
+            }
+            else if ("stillbirths".equals(yamlConfig.getSubGroup())) {
+
+                yamlConfigListGlobal.add(new YamlConfigWrapper(null, "stillbirths", null));
+
+                int count = PncLibrary.getInstance().getPncStillBornRepository().count(facts.get("base_entity_id"));
+                facts.put("baby_count_stillborn", String.valueOf(count));
+                yamlConfigListGlobal.add(getConfigItem("Number of babies stillborn: {baby_count_stillborn}", count > 0));
+            }
         }
```

Comment:
follow the structure in profile-overview for this

Reply:
make it partially similar to opd